crane/cmd.go

```diff
@@ -42,8 +43,11 @@ func configCommand(wrapped func(config Config), forceOrder bool) func(cmd *cobra
 			}
 		}
 
-		// Set target from args
-		options.target = args
+		// Set target and potentially command from args
+		if len(args) > 0 {
+			options.target = strings.Split(args[0], "+")
```

Comment:
Right, I went a bit quick there... It's just the passing of commands to the default group which is slightly verbose, but as discussed above, it's probably not a very common use case.